signature-proof-spec/spec.md

```diff
@@ -0,0 +1,329 @@
+    "proofs": [],
+    "merkleRoot": "704510b6b097aab7a0e3e76b2112174571c20439bcbc1373f1020d0221763405",
+    "salts": "W3sidmFsdWUiOiJiZmFkMTYzNTAwMzdkZTcxM2U5NTc3YzhmMjkzZTY1NGZiMTI5ODczYWE1OTM2NmM0YzViYzc1ODJiNjc1YWQ4IiwicGF0aCI6InZlcnNpb24ifSx7InZhbHVlIjoiNWExNjY5Mzg5NGY3YzYxODU2M2MwZjljM2YxOTZiODk3YjFiNDg0NzRiZTE0NzM2MTQyYzUzNmJkOTI3MzNkMCIsInBhdGgiOiJAY29udGV4dFswXSJ9LHsidmFsdWUiOiI3MjM5MzRmMDA3YmE5NWRhMWM0Y2MxN2U4MTM2NTI3YjliNGZmZGMxNDhmMTFkZDlkODRjNzdkN2E4MWFkMDI2IiwicGF0aCI6IkBjb250ZXh0WzFdIn0seyJ2YWx1ZSI6ImRlNWU5MTg3NDY4NjA3YTM0OWE1N2QzOGQ4ODAzOGY5NWE4MDY3OTM1OTcwNjk5MDY5MzI0MTljMDViN2EyN2YiLCJwYXRoIjoiQGNvbnRleHRbMl0ifSx7InZhbHVlIjoiMDI0ZjExNWQ1M2NkNDZjZjVhZmZkYmJhZmUzMjhhNDNiMGZjODJhYTc2ZTIyYWMzNjcxMzVlMzkzMGFmODE2YSIsInBhdGgiOiJpZCJ9LHsidmFsdWUiOiIwNWU1YjE2NTc5NDFkZjdiZDA0YjQ5ZmJhNjZlNDYyM2FkOGY1MjEyMmY2Yjc1MjVjY2ZjNWMxNDQwNDM1NjYzIiwicGF0aCI6InR5cGVbMF0ifSx7InZhbHVlIjoiYzAxYmQ5ODhiMDRiZDgxMzhmYzRjMTE0MDI0ZmUwNjQ2ZGI3ZThiNmExNTc4NGY3YmMzYTNiMmRjYjU2MTU1ZCIsInBhdGgiOiJ0eXBlWzFdIn0seyJ2YWx1ZSI6IjQwYjI4MTEyYjI1MGU1ZWE3MDgwMjNiMzY4MDE0OGVkNWMyYjY3NDIxNzJiYjg4Y2RjNzdkNzkxZWJhN2JmNmMiLCJwYXRoIjoidHlwZVsyXSJ9LHsidmFsdWUiOiIwNGY4OTU5YjA0MDc1OTBlNDM0MmZkZjZkYzFlYThiM2Y0YThhODQ4NTA3ZDcxYWYwOTM1MmUzYzU5NTRhMzIyIiwicGF0aCI6Imlzc3VlciJ9LHsidmFsdWUiOiI0NDY3MTYzOTQ4OGM0MGU2YThjNDc3N2MyZGViNWUxMWJhNjFhZGE2ZmI5ZWY0MmVkN2Q5NTEyYjE5MzJiZGYwIiwicGF0aCI6Imlzc3VhbmNlRGF0ZSJ9LHsidmFsdWUiOiIwZGU3ZDU2YzFkZDVkMWNmZTZjZGFjYTQ5Njg1NWZlZGI0MzQ3N2Q2NDliMTU4MDJlNzc4Yjg0NzAyZTVmZWNhIiwicGF0aCI6ImNyZWRlbnRpYWxTdWJqZWN0LmlkIn0seyJ2YWx1ZSI6IjA1YjhhMmNhYjdlMDg5MzIzYTVkMmY3ZjEwZDhjNTkxNGQ3MzJiNzQ1MWNlYzEyYmRkYzE5NTM2MGJjMDUwMDciLCJwYXRoIjoib3BlbkF0dGVzdGF0aW9uTWV0YWRhdGEudGVtcGxhdGUubmFtZSJ9LHsidmFsdWUiOiIyZTRmYWY5OGVjYjViYTQ5M2U1ZWZlOTkzZDk3ZTU4MDUwMzI5YzE1NDJiZWQwYjY0YmEyNjdkMDA3NTgzMWRmIiwicGF0aCI6Im9wZW5BdHRlc3RhdGlvbk1ldGFkYXRhLnRlbXBsYXRlLnR5cGUifSx7InZhbHVlIjoiYjU2ZTg5N2Q4MDkxNGEwNDEzM2NmNWE2ODY5ZTY0NzNmMDJkOTgyYTExODU0MGU4NDk2YzNhOTI0ZjJhMDJlOSIsInBhdGgiOiJvcGVuQXR0ZXN0YXRpb25NZXRhZGF0YS50ZW1wbGF0ZS51cmwifSx7InZhbHVlIjoiZjM0YWVmYTE0MjRhMzkzZGM5M2MwZTExOTU0ZmYxMDUxZDRiZjYzZjNlMTVlOGU4NjlkYjlhMmI2OTU4ODRhZiIsInBhdGgiOiJvcGVuQXR0ZXN0YXRpb25NZXRhZGF0YS5wcm9vZi50eXBlIn0seyJ2YWx1ZSI6ImRlYjM3MzFkZjVhYjA0YjY1MGRiYzUwNmYxMTM0NzcxYjU0ZDA3NTM1Yjc1ZGZiMzA2YWFmYzliOGJlZDgxZmIiLCJwYXRoIjoib3BlbkF0dGVzdGF0aW9uTWV0YWRhdGEucHJvb2YubWV0aG9kIn0seyJ2YWx1ZSI6IjNlODJkYTQwNDk1YzMwMjgyMGZlY2E3NDM5ZGRlZWJkNjZhYWQ1MmVkNmY5NDhhMTMzMjJhMjIyNWU4NDgzMmQiLCJwYXRoIjoib3BlbkF0dGVzdGF0aW9uTWV0YWRhdGEucHJvb2YudmFsdWUifSx7InZhbHVlIjoiYWI2MDI5OTI3ZjQ1N2E4Njk3NDlkNjAwNDc0NjNjMWQyOTk0MDZjN2ZmODc5NDkyYjJkMzg5OTE4NzMyOTkzMCIsInBhdGgiOiJvcGVuQXR0ZXN0YXRpb25NZXRhZGF0YS5pZGVudGl0eVByb29mLmlkZW50aWZpZXIifSx7InZhbHVlIjoiM2NkMGJiYjg0ZmFiZDA3NGYwOWI4YmMwYWIzODk3YzZjYzNkODdmMTkxNzcyMDM3OWNmMWRjMGJlN2FlMDBmNiIsInBhdGgiOiJvcGVuQXR0ZXN0YXRpb25NZXRhZGF0YS5pZGVudGl0eVByb29mLnR5cGUifV0=",
+    "privacy": {
+      "obfuscated": [
+        "06b0b34eb201e67f81e24f89ce1cdf2d280e1c1b11203b076a395a2d44e1dc27"
```

Comment:
Maybe we could link back to the array of hashes above? Currently this hash appears nowhere